engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -202,10 +199,8 @@ class WorkflowActor(workflowId: WorkflowId,
       goto(WorkflowFailedState)
   }
 
-  when(AbortingWorkflowState) {
-    case Event(x: EngineLifecycleStateCompleteResponse, _) =>
-      context.parent ! WorkflowAbortedResponse(workflowId)
-      goto(WorkflowAbortedState)
+  when(WorkflowAbortingState) {
+    case Event(x: EngineLifecycleActorAbortedResponse, _) => goto(WorkflowAbortedState)
```

Comment:
I think we did want to allow any `EngineLifecycleStateCompleteResponse` here - otherwise there'd be a race condition if e.g. the initializer "succeeds" but the WorkflowActor has already entered `WorkflowAbortingState`